composer.json

```diff
@@ -11,9 +11,15 @@
 			"email":    "james@digitaledgeit.com.au"
 		}
 	],
+	"repositories":[
+		{
+			"type": "git",
+			"url": "https://github.com/alexanderwanyoike/php-filesystem/"
```

Comment:
What's the reason for referencing your fork? Should that be merged in and let it reference the main digitaledgeit repo?